packages/cli/src/ResponseHelper.ts

```diff
@@ -6,7 +6,7 @@ import type { Request, Response } from 'express';
 import { parse, stringify } from 'flatted';
 import picocolors from 'picocolors';
 import { ErrorReporterProxy as ErrorReporter, NodeApiError } from 'n8n-workflow';
-
+import { Stream } from 'node:stream';
```

Comment:
we should use `Readable` instead of `Stream`, as streams can non-readable (like write-only streams).